### DISCUSSION TITLE
Entity selection - Unhide Selected button

### DISCUSSION THREAD

**Discussion by mrbubble62** [home-assistant/frontend#12950] [Configuration]
Due to a bug in OpenMQTTGateway (fixed), I had thousands of of IBEACON and MS-CDP entities created by MQTT .
After filtering (incorrectly) and selecting all, I clicked Hide Selected and promptly hid ALL my MQTT entities including the ones I still want.
As there is no Unhide Selected, we a forced to use multiple mouse clicks to enable each hidden item. This is not helping my carpal tunnel issues. Please consider making the Hide selected a toggle or add Unhide as an option.

**Comment by The00Dustin** [comment]
I recently noticed the same missing option for selected entities and almost opened an issue for this and then realized it might be more of a feature request.  I'm not sure how feature request discussions are supposed to work or what would determine whether or not the lack of a `SHOW SELECTED` option could be considered a bug when `ENABLE SELECTED`, `DISABLE SELECTED`, and `HIDE SELECTED` all exist, but if it should be considered a bug, I'll be happy to open an issue, and otherwise, please consider this an upvote for this request.

**Reply by karwosts** [reply]
There's no real organized way that feature requests "work", you just write something and maybe someone implements it, or maybe they don't :grin:

But I have added this option as PR #19123.
